### PR TITLE
Fix/profile viewing

### PR DESCRIPTION
**Pull request by karljerico**
### OVERVIEW

- Fixed profile viewing for following

### SNIPPETS
image
image
image

### CODE REVIEW

File: src/public/js/follow-unfollowButton [hunk 1]
```diff
@@ -1,19 +1,22 @@
...
     const userId = $(this).data('user');
-    const action = $(this).data('action');
     const button = $(this);
     const csrfToken = $('meta[name="csrf-token"]').attr('content');
+    const isUserFollowed = button.hasClass('btn-danger');
 
     $.ajax({
         type: 'POST',
-        url: '/users/' + userId + '/' + action,
+        url: '/users/' + userId + '/' + (isUserFollowed ? 'unfollow' : 'follow'),
```
Comment (hipe01mark) [inline src/public/js/follow-unfollowButton]: I noticed that you guys always using manual concatenation. You can use ECMAScript Concatenation.

File: src/public/js/follow-unfollowButton [hunk 2]
```diff
@@ -1,19 +1,22 @@
...
         },
-        success: function(response) {
-            button.text(action === 'follow' ? 'Unfollow' : 'Follow');
-            button.data('action', action === 'follow' ? 'unfollow' : 'follow');
-            button.toggleClass('btn-primary btn-danger');
+        success: function() {
+            button.text(isUserFollowed ? 'Follow' : 'Unfollow'); 
+            button.toggleClass('btn-primary btn-danger'); 
+
+            if (window.location.pathname === '/profile/' + userId) {
```
Comment (hipe01mark) [inline src/public/js/follow-unfollowButton]: I noticed that you guys always using manual concatenation. You can use ECMAScript Concatenation.

**Review by hipe01mark — DISMISSED** [review]
LGTM 🙇

**Review by hipe-francis — APPROVED** [review]
LGTM 🚀

**Review by hipe01mark — APPROVED** [review]
LGTM 🙇

**Pull request merged**